packages/connector-chrome/README.md

```diff
@@ -0,0 +1,74 @@
+# Chrome connector (`@sonarwhal/connector-chrome`)
+
+A connector to use Google Chrome via the [chrome debugging 
+protocol][cdp] in `sonarwhal`.
+
+## Installation
+
+First, you need to install [`sonarwhal`](https://sonarwhal.com/):
+
+```bash
+npm install sonarwhal
+```
+
+Then, install the new connector:
+
+```bash
+npm install @sonarwhal/connector-chrome
+```
+
+## Usage
+
+Configure the connector name in your [`.sonarwhalrc`][sonarwhalrc]
+configuration file:
+
+```json
+{
+    ...
+    "connector": {
+        "name": "chrome"
+    },
+    ...
+}
+```
+
+## Options
+
+The set of settings supported by chrome connector are:
+
+* `defaultProfile (boolean)`: Indicates if the browser should use the
+  default profile or create a new one. By default the value is `false`
+  so a new one is created. You might want to set it to `true` if you
+  want `sonarwhal` to have access to pages where the default profile is
+  already authenticated. This only applies for Google Chrome as
```

Comment:
I think we should drop the `This only applies for Google Chrome as...` part as it is not relevant here.